libs/feature/search/src/lib/metadata-view/state/mdview.actions.ts

```diff
@@ -1,6 +1,17 @@
+import { RecordSummary } from '@geonetwork-ui/util/shared'
 import { createAction, props } from '@ngrx/store'
 
 export const setUuid = createAction(
   '[Metadata view] Set uuid',
   props<{ uuid: string }>()
 )
+
+export const setPreview = createAction(
+  '[Metadata view] Set preview',
+  props<{ preview: RecordSummary }>()
+)
+
+export const setFull = createAction(
+  '[Metadata view] Set full metadata',
+  props<{ full: RecordSummary }>()
```

Comment:
Should be `RecordFull` maybe?

Reply:
I don't adress the typing in this PR.
We need to refine how we use `RecordSummary` `RecordFull` etc...

Reply:
No problem, I though it was a mistake